src/com/facebook/buck/cli/TestCommand.java

```diff
@@ -101,8 +111,13 @@
   @Option(name = "--code-coverage", usage = "Whether code coverage information will be generated.")
   private boolean isCodeCoverageEnabled = false;
 
-  @Option(name = "--code-coverage-format", usage = "Format to be used for coverage")
-  private CoverageReportFormat coverageReportFormat = CoverageReportFormat.HTML;
+  @Option(
+    name = "--code-coverage-format",
+    usage = "Comma separated Formats to be used for coverage",
+    handler = CoverageReportFormatsHandler.class
+  )
+  private CoverageReportFormat[] coverageReportFormats =
```

Comment:
when working with enums it's much more convenient to use `EnumSet`

Reply:
Thanks, switched to EnumSet where possible. unfortunately I were not able to make args4j option to work with anything but array though.